examples/grid_example/public/app.tsx

```diff
@@ -131,7 +119,7 @@ export const GridExample = ({ coreStart }: { coreStart: CoreStart }) => {
         <EuiPageTemplate.Section
           color="subdued"
           contentProps={{
-            css: { display: 'flex', flexFlow: 'column nowrap', flexGrow: 1 },
+            css: { flexGrow: 1 },
```

Comment:
I had to make some changes to how the panel "expand" view works, since it was going wrong with embeddables for... some reason 🫠 We can revisit this more closely in the Dashboard integration, since it will likely require custom styles in the Dashboard code